runtime/gc_trace/TgcRootScanner.cpp

```diff
@@ -74,7 +74,7 @@ const static char *attributeNames[] = {
 	"monitorlookupcaches", /* RootScannerEntity_MonitorLookupCaches */
 	"monitorlookupcachescomplete", /* RootScannerEntity_MonitorLookupCachesComplete */
 	"monitorreferenceobjectscomplete", /* RootScannerEntity_MonitorReferenceObjectsComplete */
-	"doubleMappedObjects", /* RootScannerEntity_DoubleMappedObjects */
```

Comment:
too early to remove double mapped string - the order of string table has to match the order of enums in OMR